requirements.txt

```diff
@@ -952,9 +984,10 @@ six==1.15.0 \
     #   oauth2client
     #   ppft
     #   protobuf
+    #   pyjnius
     #   python-dateutil
     #   virtualenv
-smart-open==6.0.0 \
+smart_open==6.0.0 \
```

Comment:
This is weird.  Running pip-compile locally using Python 3.8 changed this to `smart_open`, which seems correct to me, but the [CircleCI verify-requirments check](https://app.circleci.com/pipelines/github/mozilla/bigquery-etl/14333/workflows/bef41822-d4ab-4bb5-880b-ba071acb6567/jobs/116973) is insisting this should be `smart-open`.

Reply:
might be related to osx vs linux, but we should stick with `smart-open` if that's what CI gives us.

Reply:
Never mind, looks like I had pip-tools 6.4.0 installed.  Upgrading to pip-tools 6.6.0 and running pip-compile locally changes this to `smart-open` like the CI.